app/src/androidTest/java/org/mozilla/focus/helpers/TestHelper.kt

```diff
@@ -415,6 +418,14 @@ object TestHelper {
         return customTabsIntent.intent
     }
 
+    fun assertNativeAppOpens(appPackageName: String, url: String) {
+        if (isPackageInstalled(appPackageName)) {
+            Intents.intended(IntentMatchers.toPackage(appPackageName))
```

Comment:
`Intended` can throw `AssertionFailedError`, let's wrap in a `Try-Catch`

```Kotlin
try {
                intended(toPackage(appPackageName))
            } catch (e: AssertionFailedError) {
                e.printStackTrace()
            }
```

Reply:
Done